City-of-Pori/pori-kada
# PORI-527: Video header

What was done:
- Updated gulp to 4.0 and gulpfile accordingly
- Added styles for video header
- Added pause/play button
- Styles for dark grey scrim

Steps to test:
- Quickly check that gulp scripts for Kada theme compile as expected.
- Go to https://pori.dev.wunder.io/ and see that the video in video header autoplays and fills the header area correctly (if possible, check IE and Edge too)
- See that the video has no controls, is responsive and loops
- See that the pause button works as expected
- Slider caption is coloured according to dark grey scrim (**See note below! Please review this PR ignoring the control buttons for now**)
